lib/common/netcdf-check.sh

```diff
@@ -91,6 +91,7 @@ export -f _get_checker_code_dir
 # "$@" - checker suites passed
 add_checker_signature() {
     local file=$1; shift
+    local checks=$@
```

Comment:
@lbesnard Look at the list of args in the comment. You need to specify the checker suites that were passed. This has always been the case. See e.g. https://github.com/aodn/data-services/blob/66fcb8b2ef265ac89357e06ebb4e192ffb9add1b/lib/common/netcdf-check.sh#L158